Px.Utils.TestingApp/Commands/Benchmark.cs

```diff
@@ -94,7 +95,14 @@ internal void PrintResults()
             Console.WriteLine(new string('-', longestNameLength + 79));
             foreach (BenchmarkResult result in Results)
             {
-                Console.WriteLine(format, result.Name, result.MinTimeMs, result.MaxTimeMs, result.MeanTimeMs, result.MedianTimeMs, result.StandardDeviation);
+                if(result.Error)
+                {
+                    Console.WriteLine(format, result.Name, "Error", "Error", "Error", "Error", "Error");
```

Comment:
This could, perhaps, be a bit prettier? At least it doesn't leave uncertain that an error was encountered.

Reply:
Changed it to only print Error once